update-charts-dep.sh

```diff
@@ -13,7 +13,7 @@ find ./ -name "charts"| xargs rm -Rf
 find ./ -name "tmpcharts"| xargs rm -Rf
 
 declare -a charts=(
-        dependencies/backend
+        mojaloop/example-backend
         mojaloop/common
         mojaloop/admin-api-svc
         mojaloop/fspiop-transfer-api-svc
```

Comment:
Do we need to include the charts related to bof here?

Reply:
yes we do

Reply:
I added the BOF charts to the script files and pushed the commits to same PR